src/Compilers/CSharp/Test/Syntax/Parsing/MemberDeclarationParsingTests.cs

```diff
@@ -1915,17 +1933,30 @@ public void RequiredModifier_LocalNamedRequired_TopLevelStatements(CSharpParseOp
         [Fact]
         public void OperatorDeclaration_ExplicitImplementation_01()
         {
-            var error =
-                // (1,12): error CS8773: Feature 'static abstract members in interfaces' is not available in C# 9.0. Please use language version 11.0 or greater.
-                // public int N.I.operator +(int x, int y) => x + y;
-                Diagnostic(ErrorCode.ERR_FeatureNotAvailableInVersion9, "N.I.").WithArguments("static abstract members in interfaces", "11.0").WithLocation(1, 12);
+            var text = "public int N.I.operator +(int x, int y) => x + y;";
+            var classWithText = $"class C {{ {text} }}";
+            CreateCompilation(classWithText, parseOptions: TestOptions.Regular9).VerifyDiagnostics(
```

Comment:
Since we're only testing full compilation under LangVersion 9, it seems we're no longer demonstrating that a LangVersion error is not given in LangVersion preview. Is that fine? @AlekseyTs

Reply:
no.  that doesn't seem ok.  I'll add those back in.  apologies.